hooks/tk-multi-publish2/maya/publish_lod_geometry.py

```diff
@@ -228,6 +272,11 @@ def publish_files(self, task_settings, item, publish_path):
             "Exported group %s to '%s'." % (item.properties.fields["node"], publish_path)
         )
 
+        # Renaming top group name to be the asset name, in exported alembic.
+        published_abc = publish_path.replace("\\", "/")
+        asset_name = item.context.entity["name"]
```

Comment:
Shouldn't this be `asset_name = item.properties.fields["node"]` ?

Reply:
"node" here is maya node name (eg. md)

Reply:
Marking this as resolved, based on Aparna's clarification, as to why we are using item.context.entity["name"] for this case.